### PR TITLE
Feat(infos): added hostname in system information

### PR DESCRIPTION
**Pull request by Dreimus**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | When I have Prestashop in a load-balancing system, I can see the database name, but not the server hostname to identify on which server I am. This feature display the hostname on backoffice
| Type?             | new feature
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| UI Tests     | https://github.com/florine2623/testing_pr/actions/runs/8417257291
| How to test?      | Go into System Information Tab in backoffice and see the hostname
| Fixed issue or discussion?     | Fixes https://github.com/PrestaShop/PrestaShop/discussions/33941
| Sponsor company   | Evolutive

### CODE REVIEW

**Comment by Hlavtox** [comment]
@Dreimus Code style ;-) https://github.com/PrestaShop/PrestaShop/actions/runs/7919878241/job/21621615687?pr=35380

File: src/Adapter/Hosting/HostingInformation.php [hunk 1]
```diff
@@ -89,4 +89,12 @@ public function isApacheInstawebModule()
     {
         return Tools::apacheModExists('mod_instaweb');
     }
+
```
Comment (kpodemski) [inline src/Adapter/Hosting/HostingInformation.php]:
```suggestion

```

**Review by FabienPapet — APPROVED** [review]

**Review by TheoAlloin — APPROVED** [review]

File: src/Adapter/Hosting/HostingInformation.php [hunk 2]
```diff
@@ -89,4 +89,11 @@ public function isApacheInstawebModule()
     {
         return Tools::apacheModExists('mod_instaweb');
     }
+    /**
```
Comment (FabienPapet) [inline src/Adapter/Hosting/HostingInformation.php]:
```suggestion
    }

    /**
```

**Review by FabienPapet — DISMISSED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @Dreimus ,

Screenshot 2024-03-25 at 11 27 09


LGTM ✅ 
Thanks!

**Pull request merged**